LineSDK/LineSDK/LineSDKUI/SharingUI/PageViewController/PageTabView.swift

```diff
@@ -199,7 +199,16 @@ class PageTabView: UIView {
         delegate?.pageTabView(self, didSelectIndex: index)
     }
 
-    // This only update the `selectedIndex` property and update style when neccessary.
+    func tabIndex(from progress: CGFloat) -> Int {
+        let divider = 1.0 / CGFloat(tabs.count)
+        return min(Int(progress / divider), tabs.count - 1)
```

Comment:
```suggestion
        return min(Int(progress * CGFloat(tabs.count)), tabs.count - 1)
```
Maybe we could use progress times tab count directly

Reply:
Yes.